__tests__/api.tests/topics.test.js

```diff
@@ -0,0 +1,26 @@
+const request = require('supertest');
+const app = require('../../app');
+const seed = require('../../db/seeds/seed');
+const data = require('../../db/data/test-data/');
+const db = require('../../db/connection');
+
+beforeEach(() => {
+    return seed(data);
+})
+
+afterAll(() => {
+    db.end();
+})
+
+describe('topics', () => {
+    test('GET /api/topics returns with status code 200 and all the topics', () => {
+        return request(app).get('/api/topics').expect(200).then(({body: topics}) => {
+            topics.forEach(topic => {
+                expect(topic).toEqual(expect.objectContaining({
+                    slug: expect.any(String),
+                    description: expect.any(String)
+                }))
```

Comment:
Very good :) 

However, you will need to add a length check to make sure that our forEach doesn't give a false positive.